samples/project-anfield/config.ini

```diff
@@ -70,3 +70,7 @@ storage_access_key=<access key>
 [abc.domain.com] # NAS SMB Source
 domain=<domain of smb user>
 password=<password>
+
+[netapp cluster]
```

Comment:
Is this generated in autogenerate script , 

also make this single word like [netapp] , this should support both cluster and standalone correct ?

Reply:
Yeah, this will be auto generated as well. Will look like this,,
[netapp]
password=
smb_password=
Supports both Netapp types.